### PR TITLE
Primary describe cmd

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-cli/issues/1554

Screenshot from 2024-03-18 11-36-55

### CODE REVIEW

File: cmd/primarycmd/describe.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		app,
+		globalNetworkFlags,
+		false,
+		describeSupportedNetworkOptions,
+		"",
+	)
+	if err != nil {
+		return err
+	}
+	fmt.Println(logging.LightBlue.Wrap(art))
```
Comment (arturrez) [inline cmd/primarycmd/describe.go:60]: nit: add LightBlue to other ascii art in the code base

File: cmd/primarycmd/describe.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	var (
+		teleporterMessengerAddress string
```
Comment (arturrez) [inline cmd/primarycmd/describe.go:62]: nit use `:=""` instead

File: cmd/primarycmd/describe.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+		teleporterRegistryAddress  string
+	)
+	if network.Kind == models.Local {
+		extraLocalNetworkData, err := subnet.GetExtraLocalNetworkData(app)
```
Comment (arturrez) [inline cmd/primarycmd/describe.go:66]: nit: can be shortened with if extraLocalNetworkData, err := subnet.GetExtraLocalNetworkData(app); err!=nil... else 
		teleporterMessengerAddress = extraLocalNetworkData.CChainTeleporterMessengerAddress
		teleporterRegistryAddress = extraLocalNetworkData.CChainTeleporterRegistryAddress

File: cmd/primarycmd/describe.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+		if err != nil {
+			return err
+		}
+		teleporterMessengerAddress = extraLocalNetworkData.CChainTeleporterMessengerAddress
+		teleporterRegistryAddress = extraLocalNetworkData.CChainTeleporterRegistryAddress
+	} else if network.ClusterName != "" {
+		clusterConfig, err := app.GetClusterConfig(network.ClusterName)
```
Comment (arturrez) [inline cmd/primarycmd/describe.go:73]: same nit

**Review by arturrez — APPROVED** [review]

File: cmd/primarycmd/describe.go (hunk 2 above)
Comment (felipemadero) [reply cmd/primarycmd/describe.go:62]: I believe default implicit value is a preferer golang idiom

**Pull request merged**